packages/react-performance/README.md

```diff
@@ -17,7 +17,36 @@ $ yarn add @shopify/react-performance
 
 The most basic way to use the tools in this package is to record information and display it locally to the user. In practice you usually only want to do this in development so that developers can easily see performance information.
 
-#### Display data using NavigationListener
+### Adding some `<PerformanceMark />`s
+
+We will need to add some `<PerformanceMark />` components to our React application in order to help `@shopify/react-performance` determine when your application is fully loaded and React has mounted your components.
+
+```tsx
+//Cats.tsx
+import React from 'react';
+import {PerformanceMark} from '@shopify/react-performance';
+
+/**
+ * Represents an entire page of our application
+ */
```

Comment:
I wonder if this should be

```
export function Cats() {
  return (
    <>
      <section>
        <h1>Cats</h1>
        <ul>
          <li>Kokusho</li>
          <li>Moe</li>
          <li>Smoky</li>
        </ul>
      </section>
      <PerformanceMark stage="complete" id="cats" />
    </>
  );
```

so the complete will include the render of `<section />` as well. (have I been doing this all wrong actually?)

Reply:
I don't think it matters too much, but possibly slightly more accurate